committees/templates/committees/group_detail_info.html

```diff
@@ -0,0 +1,96 @@
+{% extends 'committees/group_detail_base.html' %}
+{% load static %}
+{% load to_member %}
+
+{% block content-frame-class %}
+    container
+{% endblock %}
+
+
+{% block content %}
+{{ block.super }}
+
+
+
+    <div class="row">
+        <div class="col-12 col-md-8">
+            <div style="position: relative;">
+            <div class="" style="position: absolute; top: 0; right: 0;">
+                <a href="{% url "committees:group_update" group_id=association_group.id %}"
+                   class="btn btn-outline-primary btn-sm">
+                    Edit
+                </a>
+            </div>
+            </div>
+            <div style="min-height: 2em;">
```

Comment:
```suggestion
            <div style="min-height: 2em;" class="rendered-md">
```
This class is needed for additional CSS to load. See #129